buildifier/convert_ast.go

```diff
@@ -0,0 +1,250 @@
+// This file contains functions to convert from one AST to the other.
+// Input: AST from github.com/google/skylark/syntax
+// Output: AST from github.com/bazelbuild/buildtools/build
+
+package main
```

Comment:
I think this file belongs in its own
package astconvert 
with some of these functions public